config/exporters/monitoring/grafana/base/prometheus-exporter-service-monitor.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: metrics-reader
+  namespace: appstudio-grafana-datasource-exporter
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: metrics-reader
+  namespace: appstudio-grafana-datasource-exporter
+  annotations:
+    kubernetes.io/service-account.name: metrics-reader
+type: kubernetes.io/service-account-token
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I think we should make it a role rather than a cluster role, and the binding to be a role binding. We're staying within the namespace as far as I see it. No need to potentially allow access outside of the namespace.

Reply:
This question still stands